src/components/HelperErrorText/HelperErrorText.mdx

```diff
@@ -50,42 +50,42 @@ use case of the `HelperErrorText` component is to dynamically render error text
 when a form field is invalid. In that scenario, the dynamically updated error
 text will be read by assistive technologies.
 
-The example below is shows a `TextInput` component. The `TextInput` component
+The example below shows a `TextInput` component. The `TextInput` component
 uses the `HelperErrorText` component to render the red-colored `"This is error text :("` text below the input field and that text element is referenced with
 the `aria-describedby` attribute on the HTML input element. This type of
 association is handled automatically by all DS `Form Elements` components.
 
 <Canvas of={HelperErrorTextStories.TextInputExample} />
 
-### ariaAtomic
+### ARIA Live
 
-The `ariaAtomic` prop sets the `aria-atomic` HTML attribute. This allows the
-entire DOM element to be presented to assistive technologies. When it is set to
-`false`, only additions or removals will be read by assistive technologies.
+By default, `aria-live` is set to `polite` for the `HelperErrorText` component.
+`aria-live` determines when text updates should be read by assistive technologies
+and the `"polite"` value indicates that updates should be read whenever the user
+is next idle (important, but not requiring immediate attention).
 
-By default, this prop is set to `true`.
+If desired, you can pass `aria-live="off"` or `aria-live="assertive"` instead.
 
-### ariaLive
+The `"off"` value indicates that updates should not be announced. When this
+value is passed, the `aria-live` attribute will not be added at all.
 
-The `ariaLive` prop sets the `aria-live` HTML attribute. This determines the
-priority of when text updates in this component should be read to users by
-assistive technologies. The possible values are the same as the native HTML
-`aria-live` attribute: `"off"`, `"polite"`, and `"assertive"`.
-
-The `"off"` value indicates that updates should not be announced and when this
-value is passed, the `aria-live` attribute will not be added at all. The
-`"polite"` value indicates that updates will be announced at the next available
-time slot. The `"assertive"` value indicates that updates should be announced
-immediately, but this is not recommended as it can interruptive to the user.
+The `"assertive"` value indicates that updates should be announced
+immediately. This is generally not recommended as it can interruptive
+to the user.
 
 In order to meet the accessibility guidelines for dynamic content updates, this
 attribute is set in the wrapper `<div>` element that is always rendered by the
 `HelperErrorText` component.
 
-By default, this prop is set to `"polite"`.
-
 <Canvas of={HelperErrorTextStories.AriaControls} />
 
+### ARIA Atomic
+
+When `aria-atomic` is set to `true` within an ARIA live region, the entire DOM
+element is presented to assistive technologies. By default, `aria-atomic` is
+set to `true` for the `HelperErrorText` component. If desired, you can pass
+`aria-atomic={false}` instead so that only the changed node is presented.
```

Comment:
I saw that @bigfishdesign13 made a comment about "additions and removals". "only the changed node" is helpful but that update might not always be a node. It's possible that the only change inside an element with aria-atomic is a string, such as
```
<div aria-atomic="false">
 This is some string that doesn't change but ${variableThatUpdatesConstantly} does change.
</div>
```

In this case, with `aria-atomic={false}`, only the string value from `variableThatUpdatesConstantly` will be read by screen readers. So that's the "additions and removals" from before.